mifosng-android/hs_err_pid13995.log

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
Add *.log to .gitignore so that these kind of files are not tracked by and checked into the version control.